src/test/js/multibranch/testResults.js

```diff
@@ -40,8 +40,11 @@ module.exports = {
         blueRunDetailsPage.clickTab('tests');
 
         // Expand the test.
-        browser.useXpath().click('//div[@class="result-item-head"]');
-
-        browser.useXpath().waitForElementVisible('//div[@class="test-console"]/h4')
+        browser.useXpath().waitForElementVisible('//span[contains(text(), "StdOut")]');
```

Comment:
@scherler I disagree in this case. I think that we should add commonly used things, but this is a very specific xpath just for this test.